src/plugins/controls/public/controls/timeslider_control/components/time_slider_popover_content.tsx

```diff
@@ -27,29 +29,62 @@ interface Props {
   compressed: boolean;
 }
 
-export function TimeSliderPopoverContent(props: Props) {
-  const rangeInput = props.isAnchored ? (
+export function TimeSliderPopoverContent({
+  isAnchored,
+  setIsAnchored,
+  value,
+  onChange,
+  stepSize,
+  ticks,
+  timeRangeMin,
+  timeRangeMax,
+  compressed,
+}: Props) {
+  const [displayedValue, setDisplayedValue] = useState<Timeslice>(value ?? [0, 0]);
```

Comment:
Value is a required prop, so I don't think we need a fallback?

```suggestion
  const [displayedValue, setDisplayedValue] = useState<Timeslice>(value);
```